src/tests/ie_test_utils/common_test_utils/crash_handler.hpp

```diff
@@ -15,10 +15,16 @@ namespace CommonTestUtils {
 
 extern jmp_buf env;
 
+enum JMP_STATUS { ok = 0, anyError = 1, alarmErr = 2 };
+
 class CrashHandler {
+private:
+    static unsigned int MAX_TEST_WORK_TIME;
 public:
     CrashHandler();
     ~CrashHandler();
+    static void SetUpTimeout(unsigned int timeout);
+    void StartTimer();
```

Comment:
`static`?

Reply:
no, it is need to turn off the timer, I put it to destructor